### PR TITLE
tests: Add label focus

### PR DESCRIPTION
**Pull request by oshoval**
**What this PR does / why we need it**:
Allow to use `KUBEVIRT_LABEL_FILTER` in order to focus certain labels.
usage: `export KUBEVIRT_LABEL_FILTER="(x)"`

Note: the global [labels](https://github.com/kubevirt/kubevirt/pull/10987/files#diff-ce25e369a9df9edc525425cc144fc8c40599fff3cc1e5058b64336d71af67162R437-R450) will be still added.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
None
```

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -381,8 +381,8 @@ add_to_label_filter() {
   fi
 }
 
-# Set label_filter only if KUBEVIRT_E2E_FOCUS and KUBEVIRT_E2E_SKIP are not set.
-if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
+# Set label_filter only if KUBEVIRT_E2E_FOCUS, KUBEVIRT_E2E_SKIP and label_filter are not set.
```
Comment (0xFelix) [inline automation/test.sh:384]:
```suggestion
# Set label_filter only if KUBEVIRT_E2E_FOCUS, KUBEVIRT_E2E_SKIP and KUBEVIRT_LABEL_FILTER are not set.
```

Comment (oshoval) [reply automation/test.sh]: done

**Review by 0xFelix — COMMENTED** [review]
What about L437? This would still modify the label filter given by the environment. I'd check if `KUBEVIRT_LABEL_FILTER` is empty in L450 and if not then I'd override `label_filter` regardless of what was set before.

**Comment by oshoval** [comment]
> **What this PR does / why we need it**: Allow to use `KUBEVIRT_LABEL_FILTER` in order to focus certain labels. usage: `export KUBEVIRT_LABEL_FILTER="(x)"`
> 
> Note: the global [labels](https://github.com/kubevirt/kubevirt/pull/10987/files#diff-ce25e369a9df9edc525425cc144fc8c40599fff3cc1e5058b64336d71af67162R436-R449) will be still added.

Left deliberately, please see note on the PR body
those global labels are needed regardless usually, because it is for unsupported features / QUARANTINE (can be omitted by setting `KUBEVIRT_QUARANTINE`) and single node restrictions that needed for single node

**Review by 0xFelix — APPROVED** [review]
Thanks!

/lgtm

**Review by enp0s3 — COMMENTED** [review]
/approve

@oshoval Thank you!

**Pull request merged**